Source/WebKit/NetworkProcess/webtransport/NetworkTransportReceiveStream.h

```diff
@@ -25,19 +25,45 @@
 
 #pragma once
 
+#include <wtf/ObjectIdentifier.h>
+#include <wtf/RefCounted.h>
 #include <wtf/TZoneMalloc.h>
 #include <wtf/WeakPtr.h>
 
+#if PLATFORM(COCOA)
+#include <Network/Network.h>
+#include <wtf/RetainPtr.h>
+#endif
+
+namespace WebKit {
+struct WebTransportStreamIdentifierType;
+using WebTransportStreamIdentifier = LegacyNullableObjectIdentifier<WebTransportStreamIdentifierType>;
```

Comment:
I'm not a reviewer of course--but saw this and had a question/suggestion. 

Feel free to correct me if I'm wrong: I believe we're trying to move away from using `LegacyNullableObjectIdentifier`. Is there a specific reason we need it here? Would it be possible to use `ObjectIdentifier` instead? I was just curious so I thought I'd ask. Thank you!

Reply:
You're right, but this is just another declaration of the existing WebTransportStreamIdentifier, which should be changed in a separate PR.

Reply:
Got it--that makes sense!